lib/cpp/hawkes/inference/hawkes_cumulant.cpp

```diff
@@ -126,3 +124,66 @@ double HawkesCumulant::compute_E_ijk(ulong r, ulong i, ulong j, ulong k,
   res /= (*end_times)[r];
   return res;
 }
+
+HawkesTheoreticalCumulant::HawkesTheoreticalCumulant(int dim) : d(dim) {
+  Lambda = SArrayDouble::new_ptr(dim);
```

Comment:
1.  is `Lambda` the best name?
2.  variables names don't typically start with upper case

Reply:
The variables `Lambda`, `C`, `Kc`, and `R` represent first integrated cumulant, second integrated cumulant, (reduced) third integrated cumulant, and inverse of $I - G$ (where $G$ is the matrix of $L^1$-norms of kernel functions), respectively.  Their names are directly taken from the math symbols used in the [paper](https://arxiv.org/abs/1607.06333): $\Lambda$, $C$, $K$, and $R$, respectively. 

I agree that the upper case is uncommon. I can change the name as follows:
`Lambda` -> `first_cumulant`
`C` -> `second_cumulant`
`Kc` -> `third_cumulant`
`R` -> `g_geometric`

Reply:
Done here: 11353f445b59009527ca12464a5eab988bc2b844